content/yaml-quick-start/building-a-unity-app.md

```diff
@@ -189,6 +189,20 @@ public static class BuildScript
         Debug.Log("Built iOS");
     }
 
+    [MenuItem("Build/Build Windows")]
+    public static void BuildWindows()
+    {
+        BuildPlayerOptions buildPlayerOptions = new BuildPlayerOptions();
+        buildPlayerOptions.locationPathName = "win/" + Application.productName + ".exe";
+        buildPlayerOptions.target = BuildTarget.StandaloneWindows;
+        buildPlayerOptions.options = BuildOptions.None;
+        buildPlayerOptions.scenes = GetScenes();
+
+        Debug.Log("Building Windows");
+        BuildPipeline.BuildPlayer(buildPlayerOptions);
+        Debug.Log("Built Windows");
+    }
+
```

Comment:
This page is a quick start for building 'Building Unity **mobile** apps' for Android and iOS. 

Not sure how adding the script for building a Windows desktop fits in here.

Reply:
We don't have any docs for Unity & Windows. Even for the Steam publishing, where Windows is the main platform. If not here, then we need some other place for that.

Reply:
Let's put it here for now. We can possibly come up with a better solution at a later date.